src/modules/healtcheck/healtcheck.service.ts

```diff
@@ -0,0 +1,30 @@
+import { Injectable} from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { MarketEntity } from '../markets/entities/market.entity';
+import { UkefNumber as Number } from '../numbers/entities/number.entity';
+
+@Injectable()
+export class HealtcheckService {
+
+  constructor(
+    @InjectRepository(Number, "mssql-number-generator")
+      private readonly numberRepository: Repository<Number>,
```

Comment:
`number` in lowercase please - to indicate the response type

Reply:
Renamed Number class to UkefId. Module was also renamed, this includes folder and filename change.